src/main/java/nl/tudelft/jpacman/board/Board.java

```diff
@@ -30,7 +31,7 @@
 	 * Whatever happens, the squares on the board can't be null.
 	 * @return false if any square on the board is null.
 	 */
-	protected final boolean invariant() {
+	protected final boolean invariant(@UnderInitialization(Board.class) Board this) {
```

Comment:
I have [improved](https://github.com/typetools/checker-framework/commit/20765ab1cc32ff0fe697a44fa753b069cd7b80d2) the documentation of `@UnderInitialization` in hopes of making it easier to use.  The improvements will appear in the online manual when we make the May 1 release of the Checker Framework.